.github/workflows/sycl-cts-linux-cpu-latest-nightly.yml

```diff
@@ -0,0 +1,20 @@
+name: SYCL-CTS on Linux&CPU with the latest nightly
+run-name: SYCL-CTS - ${{ github.ref_name }}
+
+on:
+  push:
+    tags:
+      - nightly-*
+
+jobs:
+  run:
+    uses: ./.github/workflows/sycl-linux-run-tests.yml
+    with:
+      name: SYCL-CTS
+      runner: '["debug"]'
+      image: ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:latest
+      image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+      target_devices: opencl:cpu
+      tests_selector: cts
+      install_drivers: false
+      ref: ${{ github.sha }}
```

Comment:
nit: isn't it the default?

Reply:
Looks like no, see https://github.com/intel/llvm/actions/runs/8007137309